docs/topics/Gateway.md

```diff
@@ -673,11 +673,20 @@ A user's presence is their current state on a guild. This event is sent when a u
 
 ###### Game Structure
 
-| Field | Type | Description | Present |
-|-------|------|-------------|---------|
-| name | string | the game's name | Always |
-| type | integer | see [Activity Types](#DOCS_GATEWAY/game-object-activity-types)  | Always |
-| url | ?string | stream url, is validated when type is 1  | When type is 1 |
+| Field | Type | Description |
+|-------|------|-------------|
+| name | string | the game's name |
+| type | integer | see [Activity Types](#DOCS_GATEWAY/game-object-activity-types)  |
+| url? | ?string | stream url, is validated when type is 1  |
+| timestamps? | see [timestamps](#DOCS_GATEWAY/game-object-game-timestamps) | Unix timestamps for start and/or end of the game |
```

Comment:
Shouldn't these should be `xyz object`, linked to the object table, instead of `see xyz`.

Reply:
Yes please 👍